clru/VERSION

```diff
@@ -1 +1 @@
-0.1.2
+0.1.3
```

Comment:
Is `0.1.2` already released? In that case, I think you should update the Changelog entry to the new version

Reply:
Nope, last released version is 0.1.1

Reply:
Why the bump to `0.1.3` then? :thinking:

Reply:
Huh.I thought I had reverted that.